src/analyses/submitContractRequest.ts

```diff
@@ -0,0 +1,28 @@
+import { compileContract } from '../util/compileContract'
+
+
+//TODO: ADD EXTRA PARAM CONTRACTNAME
+export async function submitContractRequest(contractName: string, path: string) {
+
+    const { solName, contractContent, bytecode } = compileContract(path)
```

Comment:
Are we removing compilation?

We can probably change this method to be a source code only request, without any compilation. WDYT?